src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -3061,3 +3059,37 @@ instruct extractL(iRegLNoSp dst, vReg src, immI idx,  pRegGov pTmp, rFlagsReg cr
   %}
   ins_pipe(pipe_slow);
 %}
+
+instruct extractF(vRegF dst, vReg src, immI idx,  pRegGov pTmp, rFlagsReg cr)
+%{
+  predicate(UseSVE > 0);
+  match(Set dst (ExtractF src idx));
+  effect(TEMP pTmp, KILL cr);
+  ins_cost(2 * SVE_COST);
+  format %{ "movzw rscratch1, $idx\n\t"
+            "sve_whilele $pTmp, S, zr, rscratch1\n\t"
+            "sve_lastb $dst, S, $pTmp, $src\n\t" %}
+  ins_encode %{
+    __ movzw(rscratch1, (int)($idx$$constant));
+    __ sve_whilele(as_PRegister($pTmp$$reg), __ S, zr, rscratch1);
+    __ sve_lastb(as_FloatRegister($dst$$reg), __ S, as_PRegister($pTmp$$reg), as_FloatRegister($src$$reg));
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+instruct extractD(vRegD dst, vReg src, immI idx,  pRegGov pTmp, rFlagsReg cr)
+%{
+  predicate(UseSVE > 0);
+  match(Set dst (ExtractD src idx));
+  effect(TEMP pTmp, KILL cr);
+  ins_cost(2 * SVE_COST);
+  format %{ "movzw rscratch1, $idx\n\t"
+            "sve_whilele $pTmp, D, zr, rscratch1\n\t"
+            "sve_lastb $dst, D, $pTmp, $src\n\t" %}
```

Comment:
Thanks for the implementation! It mostly looks good to me. Just a trivial comment: add a comment at the end like "extract from vector(D)" like what you have done for` extractB/S` ?

Reply:
OK.I will fix that.